### PR TITLE
Finish recipe instructions

### PR DESCRIPTION
**Pull request by ElreyB**

### CODE REVIEW

**Comment by ElreyB** [comment]
@obartra 
Just finish this one. 
QUESTION
It looks like you can use HTML in Markdown. Is that preferred or does it matter?

**Comment by obartra** [comment]
Do you mean adding a README.html instead of a .md one? Not sure what you mean

File: Hearty-Vegetable-lasagana.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
 #**Hearty Vegetable Lasagna**
 
-![](http://www.feastsfromthepantry.com/wordpress/wp-content/uploads/2011/08/tumblr_lq8zjnW2T71qgmbh4.jpg)
+![](http://images.media-allrecipes.com/userphotos/560x315/620560.jpg  "Lasagana")
```
Comment (obartra) [inline Hearty-Vegetable-lasagana.md:3]: Looks like an extra "Lasagna"

If you want to show "Lasagna" as fallback text the syntax is:

```markdown
[Lasagana](http://images.media-allrecipes.com/userphotos/560x315/620560.jpg)
```

File: Hearty-Vegetable-lasagana.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
 
-#### __*Prep Time:*__ 25 minutes    
-### __*Ready in:*__ 1 Hour 40 minutes
-### __*Cook Time:*__ 1 Hour        
-### __*Serving:*__ 12
+Ready in|Prep Time |Cook time|Serving
```
Comment (obartra) [inline Hearty-Vegetable-lasagana.md:5]: Much better as a table :+1:

File: Hearty-Vegetable-lasagana.md [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+| | |
+|---|---|
+1 (16 ounce) package lasagna noodles | 2 (26 ounce) jars pasta sauce
+1 pound fresh mushrooms, sliced | 1 teaspoon dried basil
+3/4 chopped green pepper | 1 (15 ounce) container part-skim ricotta cheese
+3/4 cup chopped onion | 4 cups shredded mozzarella cheese
+3 cloves garlic, minced | 2 eggs
+2 tablespoons vegetable oil | 1/2 cup greated Parmesan cheese 
+
+##**Directions**
```
Comment (obartra) [inline Hearty-Vegetable-lasagana.md:21]: In general you don't want to style the titles, the idea is that '#' will indicate top level (`<h1>`), `##` will be an `<h2>` and so on so we shouldn't add `**` to titles

**Comment by obartra** [comment]
Just commented on a couple minor issues, I would recommend fixing the image with the extra text, everything else is fine

**Pull request merged**